app/views/users/show.html.erb

```diff
@@ -1 +1,31 @@
-<h2>User information and posts</h2>
\ No newline at end of file
+<div class="users-container">
+  <div class="user-card">
+    <div class="user-photo">
+      <%= image_tag @user.photo, alt: "#{ @user.name }'s photo" %>
+    </div>
+    <div class="user-details">
+      <%= link_to @user.name, user_path(@user) %>
+      <p>Number of posts: <%= @user.posts_counter %></p>
+    </div>
+  </div>
+</div>
+
+<div class="user-bio">
+  <h2>Bio</h2>
+  <p><%= @user.bio %></p>
+</div>
+
+<div class="user-posts">
+  <% @user.three_most_recent_posts.each do |post| %>
+    <div class="post">
+      <h3><%= link_to post.title, post_path(post) %></h3>
```

Comment:
[OPTIONAL BUT RECOMMENDED] Please make sure you use the correct. Still, for now, you are using the wrong path so kindly replace `post_path` with `user_post_path` because when I try to go to the `user/1` route it given me the following error and in this error, it mentions your wrong path which is an undefined method so kindly fix this issue as well :slightly_smiling_face: 

![Screenshot from 2023-12-17 00-21-17](https://github.com/gilberthappi/Blog-app/assets/45226848/16a214c4-e3c9-41ae-afe6-fe36989b42a9)

Reply:
Okey Thanks